### PR TITLE
Shimmer - Learn - Add High Level Shimmer EVM Content

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

Adding content to promote Shimer EVM in the Lean section.

## Links to any relevant issues

resolves https://github.com/iotaledger/guild-devx/issues/55

## Type of change

- Documentation 

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work

TODO:
- [x] add keywords and metatags
- [x] review content 
- [x] edit
- [x] add internal links
- [x] check about rng and mev content
- [ ] request and add images
- [ ] add to front page carousel

### CODE REVIEW

**Review by Dr-Electron — CHANGES_REQUESTED** [review]

File: shimmer/learn/sidebars.ts [hunk 1]
```diff
@@ -32,6 +32,43 @@ module.exports = {
         'glossary',
       ],
     },
+    {
+      type: 'category',
+      label: 'Shimmer EVM',
+      link: {
+        type: 'doc',
+        id: 'shimmer-EVM/introduction',
+      },
```
Comment (lucas-tortora) [inline shimmer/learn/sidebars.ts:41]:
```suggestion
```

File: shimmer/learn/sidebars.ts [hunk 2]
```diff
@@ -32,6 +32,43 @@ module.exports = {
...
+      collapsed: true,
+      items: [
+        'shimmer-EVM/introduction',
```
[the remark this answers is not in the document]
Comment (lucas-tortora) [reply shimmer/learn/sidebars.ts:40]: I'll remove the link then

File: shimmer/learn/shimmer-EVM/composability.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+:::note tl;dr;
+The Shimmer network enables full customization of smart contract chains while retaining interoperability and
+composability between them, allowing for near-limitless capacity and horizontal scaling.
+:::
+
+## Tailoring Smart Contract Chains to Your Needs
+
+With the Shimmer network, you can customize every aspect of your smart contract chain, including virtual machines, smart
+contract types, languages, consensus models, validators, fees, and more. The only requirement to maintain composability
```
[the remark this answers is not in the document]
Comment (lucas-tortora) [reply shimmer/learn/shimmer-EVM/composability.md:26]: According to the source article, yup.

[the remark this answers is not in the document]
Comment (Dr-Electron) [reply shimmer/learn/shimmer-EVM/composability.md:26]: Ok, I asked the team to clarify. But this is no blocker for merging :)

**Review by Dr-Electron — APPROVED** [review]

**Pull request merged**